network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_card.html

```diff
@@ -12,13 +12,18 @@
 each blog page's dominant tag)
 
 {% block tags %}
-  {% with topic=page.specific.topics.first %}
-    {% if topic %}
-      {% localized_version topic as localized_topic %}
-      {% get_root_or_page as parent_page %}
+  {% with topics=page.specific.topics.all %}
+    {% if topics %}
```

Comment:
**Suggestion** Since `page.specific.topics.all` is a `QuerySet` you don't need the `if` block. `all` will always return a `QuerySet`. That `QuerySet` may be empty but it will be a `QuerySet`. Iterating over an empty `QuerySet` will just do nothing. 

Usually you only need the `if` before a `for` block if you want to use a `<ul>` around the `<li>` elements that you are rendering in the loop.

Reply:
Nice catch Tibor! I think this will help readability. I have updated this and then marked it as resolved